validation/validation.go

```diff
@@ -326,8 +326,8 @@ type UPlugin struct {
 }
 
 type Plugin struct {
-	BIsBasePlugin *bool  `json:"bIsBasePlugin"`
-	BIsOptional   *bool  `json:"bIsOptional"`
+	BasePlugin *bool  `json:"BasePlugin"`
+	Optional   *bool  `json:"Optional"`
```

Comment:
nit: Looks like this struct used to be aligned. No idea if it matters.

Reply:
Yeah, it does, go is pretty strict about formatting (which is why the lint failed too)

Reply:
Seems it failed on something else, or am I not reading it right?

run golangci-lint
  Running [/home/runner/golangci-lint-1.49.0-linux-amd64/golangci-lint run --out-format=github-actions --timeout 5m] in [] ...
  Error: File is not `gofumpt`-ed (gofumpt)
  
  Error: issues found
  Ran golangci-lint in 57580ms
  
  Lint runs amount to this:
  
  chris@Chris-PC:/mnt/a/workspace/smr-api$ golangci-lint run --fix
WARN [config_reader] The configuration option `run.skip-files` is deprecated, please use `issues.exclude-files`.
WARN [config_reader] The configuration option `run.skip-dirs` is deprecated, please use `issues.exclude-dirs`.
WARN [config_reader] The configuration option `linters.govet.check-shadowing` is deprecated. Please enable `shadow` instead, if you are not using `enable-all`.
generated/custom_models.go:1: : # github.com/satisfactorymodding/smr-api/generated
generated/custom_models.go:24:20: undefined: CompatibilityInfoInput
generated/custom_models.go:25:21: undefined: UpdateUserMod (typecheck)
package generated
models/filters.go:13:2: could not import github.com/satisfactorymodding/smr-api/generated (-: # github.com/satisfactorymodding/smr-api/generated
generated/custom_models.go:24:20: undefined: CompatibilityInfoInput
generated/custom_models.go:25:21: undefined: UpdateUserMod) (typecheck)
        "github.com/satisfactorymodding/smr-api/generated"
        ^
redis/redis.go:21:2: could not import github.com/satisfactorymodding/smr-api/generated (-: # github.com/satisfactorymodding/smr-api/generated
generated/custom_models.go:24:20: undefined: CompatibilityInfoInput
generated/custom_models.go:25:21: undefined: UpdateUserMod) (typecheck)
        "github.com/satisfactorymodding/smr-api/generated"
        ^
db/postgres/mod.go:13:2: could not import github.com/satisfactorymodding/smr-api/generated (-: # github.com/satisfactorymodding/smr-api/generated
generated/custom_models.go:24:20: undefined: CompatibilityInfoInput
generated/custom_models.go:25:21: undefined: UpdateUserMod) (typecheck)
        "github.com/satisfactorymodding/smr-api/generated"
        ^
cmd/api/serve.go:15:2: undefined: smr (typecheck)
        smr.Start()
        ^
tests/utils.go:20:9: undefined: smr (typecheck)
        ctx := smr.Initialize(context.Background())
               ^
tests/utils.go:40:2: undefined: smr (typecheck)
        smr.Migrate(ctx)
        ^
chris@Chris-PC:/mnt/a/workspace/smr-api$